Ports/stpuzzles/patches/0002-Add-icon-files.patch

```diff
@@ -0,0 +1,2213 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Gunnar Beutner <gunnar@beutner.name>
+Date: Sun, 6 Jun 2021 19:19:04 +0200
+Subject: [PATCH] Add icon files
+
+These are statically generated because generating them during the build
+would require building the games for the host system.
```

Comment:
We should really look into making a separate host build as part of the port and then copy over the files instead of having a 2200 line patch sitting here.

Reply:
I tried building stpuzzles on MacOS, Ubuntu and Windows today, and so far I've only been able to generate the icons on Ubuntu and they were not even in the right format for serenity, so a host build is out of scope right now.

But you are right about the long ugly patch. The other options are hosting the files in the serenity repo or fetching them from somewhere. I'm open to change this if necessary.